src/js/App/RootApp/RootApp.js

```diff
@@ -12,11 +12,73 @@ import useQuickstartsStates from '../QuickStart/useQuickstartsStates';
 import { populateQuickstartsCatalog } from '../../redux/actions';
 import { LazyQuickStartCatalog } from '../QuickStart/LazyQuickStartCatalog';
 import useHelpTopicState from '../QuickStart/useHelpTopicState';
+import Notifications from '../Notifications';
+
+const useEvents = () => {
+  console.log('Use events was for some reason triggered');
+  const events = useRef({});
+  const conn = useRef();
+  const activeModule = useSelector(({ chrome: { activeModule } }) => activeModule);
+  const dispatch = useDispatch();
+  function handleEvent(payload) {
+    if (payload.type === 'new-notification') {
+      dispatch({ type: 'add-notification', payload });
+    } else if (payload.type === 'invalidate') {
+      Object.values(events.current)
+        .flat()
+        .forEach((cb) => {
+          cb(payload);
+        });
+    }
+    console.log(events);
+  }
+
+  /**
+   * Establish connection
+   */
+  useEffect(() => {
+    const x = new WebSocket('ws://localhost:8080/ws');
```

Comment:
I'd rather see a proxy config in webpack for websockets.

Reply:
I have to do some research on this. I am not sure how the webpack proxy works with WS. Not sure if our implementation will work properly.